manifests/server/config.pp

```diff
@@ -35,6 +35,13 @@
     mode   => '0700',
   }
 
+  file { $mcollective::ssldir:
+    ensure => 'directory',
+    owner  => 'root',
+    group  => '0',
+    mode   => '0755',
+  }
```

Comment:
oh, we're using contain?
in that case we don't support ***ooooooooold*** puppet versions anyway, and we're fine :D